src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/AssessmentOrgsRepository.cs

```diff
@@ -90,36 +93,32 @@ public IEnumerable<Organisation> GetOrganisationsByStandardId(string standardId)
                 throw new ApplicationException($"Failed query organisations by standard id");
             }
 
-            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date);
+            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date && (x.EffectiveTo >= DateTime.UtcNow.Date || x.EffectiveTo == null)).ToList();
```

Comment:
results.Documents.Where(x => ((x.EffectiveFrom != null) || x.EffectiveFrom.Date <= DateTime.UtcNow.Date) && (!x.EffectiveTo.HasValue || x.EffectiveTo.Value >= DateTime.UtcNow.Date)).ToList(); ?

looking at the excel file "Register - Standards" effective from can be null as well and we consider that scenario as not valid standard document.